apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ClaimVerification/messages.ts

```diff
@@ -0,0 +1,13 @@
+import { defineMessage } from "react-intl";
+
+export const priorityTitle = defineMessage({
+  defaultMessage: "Priority status",
+  id: "66vzXk",
+  description: "Title for a candidates priority status",
+});
+
+export const veteranTitle = defineMessage({
+  defaultMessage: "Veteran status",
+  id: "fI87Ov",
+  description: "Title for a candidates veteran status",
+});
```

Comment:
Can reuse `OVWo88`

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/19998103-be43-40a1-88df-e69e1523e589)

Reply:
also merged in [combine messages for re-use](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10502/commits/9d76128b02d7ee705bea95ad5e3d14745b358b13)